bindings/rust/s2n-tls/src/client_hello.rs

```diff
@@ -222,115 +398,16 @@ impl fmt::Debug for ClientHello {
         let session_id = self.session_id().map_err(|_| fmt::Error)?;
         let session_id = hex::encode(session_id);
         let message_head = self.raw_message().map_err(|_| fmt::Error)?;
-        let mut hash = Vec::new();
-        self.fingerprint_hash(FingerprintType::JA3, &mut hash)
-            .map_err(|_| fmt::Error)?;
```

Comment:
I'm disappointed that you couldn't figure out a way to keep this, but we can always add it back later.